packages/commonwealth/client/scripts/views/pages/Community/Members/CommunityMembersPage.tsx

```diff
@@ -92,7 +103,9 @@ const CommunityMembersPage = () => {
         groups: (groups || [])
           .filter((g) =>
             (g.members || []).find(
-              (x) => x?.address?.address === p.addresses?.[0]?.address,
+              (x) =>
+                x?.address?.address === p.addresses?.[0]?.address &&
+                !x.reject_reason,
```

Comment:
I've noticed this filter function repeated a few times. Should it be moved out into a dedicated function?

Since time is slim, not a biggie right now.

Reply:
I agree -- to save time i think we can do a followup for it